code/__DEFINES/icon_smoothing.dm

```diff
@@ -21,6 +21,45 @@ DEFINE_BITFIELD(smoothing_flags, list(
 	"SMOOTH_OBJ" = SMOOTH_OBJ,
 ))
 
+//Redefinitions of the diagonal directions so they can be stored in one var without conflicts
+#define NORTH_JUNCTION NORTH //(1<<0)
+#define SOUTH_JUNCTION SOUTH //(1<<1)
+#define EAST_JUNCTION EAST  //(1<<2)
+#define WEST_JUNCTION WEST  //(1<<3)
+#define NORTHEAST_JUNCTION (1<<4)
+#define SOUTHEAST_JUNCTION (1<<5)
+#define SOUTHWEST_JUNCTION (1<<6)
+#define NORTHWEST_JUNCTION (1<<7)
+
+DEFINE_BITFIELD(smoothing_junction, list(
+	"NORTH_JUNCTION" = NORTH_JUNCTION,
+	"SOUTH_JUNCTION" = SOUTH_JUNCTION,
+	"EAST_JUNCTION" = EAST_JUNCTION,
+	"WEST_JUNCTION" = WEST_JUNCTION,
+	"NORTHEAST_JUNCTION" = NORTHEAST_JUNCTION,
+	"SOUTHEAST_JUNCTION" = SOUTHEAST_JUNCTION,
+	"SOUTHWEST_JUNCTION" = SOUTHWEST_JUNCTION,
+	"NORTHWEST_JUNCTION" = NORTHWEST_JUNCTION,
+))
+
+#define ALL_JUNCTION_DIRECTIONS list(\
+	NORTH_JUNCTION,\
+	SOUTH_JUNCTION,\
+	EAST_JUNCTION,\
+	WEST_JUNCTION,\
+	NORTHEAST_JUNCTION,\
+	SOUTHEAST_JUNCTION,\
+	SOUTHWEST_JUNCTION,\
+	NORTHWEST_JUNCTION\
+	)
+
+#define NO_ADJ_FOUND 0
+#define ADJ_FOUND 1
+#define NULLTURF_BORDER 2
```

Comment:
what are these

Reply:
They are for the icon smoothing system, I didn't add them, I just moved them into the proper defines folder.